objc/objc_runtime_darwin.go

```diff
@@ -38,13 +38,19 @@ var (
 )
 
 func init() {
-	objc := purego.Dlopen("/usr/lib/libobjc.A.dylib", purego.RTLD_GLOBAL)
-	if err := purego.Dlerror(); err != "" {
-		panic("objc: " + err)
+	objc, err := purego.Dlopen("/usr/lib/libobjc.A.dylib", purego.RTLD_GLOBAL)
+	if err != nil {
+		panic("objc: " + err.Error())
+	}
+	objc_msgSend_fn, err = purego.Dlsym(objc, "objc_msgSend")
+	if err != nil {
+		panic("objc: " + err.Error())
 	}
-	objc_msgSend_fn = purego.Dlsym(objc, "objc_msgSend")
 	purego.RegisterFunc(&objc_msgSend, objc_msgSend_fn)
-	objc_msgSendSuper2_fn = purego.Dlsym(objc, "objc_msgSendSuper2")
+	objc_msgSendSuper2_fn, err = purego.Dlsym(objc, "objc_msgSendSuper2")
+	if err != nil {
+		panic("objc: " + err.Error())
```

Comment:
@hajimehoshi I think this error should be wrapped. Do you agree?

Reply:
I don't understand. Isn't err already a DlError?

Reply:
Yes but if we wanted the context that this error happens in package objc. Otherwise, just panic with the error

Reply:
I don't understand your suggestion yet. Are you suggesting to create a new error type for Objc? Even if so, this line would not change, right?

Reply:
Was this resolved?

![image](https://user-images.githubusercontent.com/16950/221001334-a06ad3d4-9fa9-41ac-9d2a-6512245d6685.png)